### DISCUSSION TITLE
关于connect握手结束和数据发送顺序先后问题

### DISCUSSION THREAD

**Discussion by yangqingyuan** [alibaba/xquic#84] [Q&A] [answered]
从demo的log来看，握手完毕之前就已经发送数据了，是这样吗？

![image](https://user-images.githubusercontent.com/6867757/153848828-b9a79b0b-ab6a-49ce-b587-7c4c4f9cb061.png)

**Comment by Kulsk** [comment, marked as answer]
The 0-RTT it is!

**Reply by yangqingyuan** [reply]
get it
